sdk/identity/identity/test/manual-integration/AzureFunctions/arm-template.json

```diff
@@ -0,0 +1,236 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vaults_name": {
+      "defaultValue": "kaghiya-standard-kv",
```

Comment:
This won't work, since it needs to be globally unique. 

Instead, we should do the following:
- Move everything other than baseName (see above) to variables instead of parameters
- "vaultName": "[concat(parameters('baseName'), 'kv']"
- "appName": "[concat(parameters('baseName'), 'fnapp']"

Etc...